section3/exercises/hashes.rb

```diff
@@ -86,16 +111,42 @@
   }
 ]
 
-puts posts
+# puts posts
 puts posts[0]
 
 
-# The code snippet above shows an Array with 2 elements. Each element in an 
-# Object Literal. Each of those Object Literals has 4 key-value pairs. This may LOOK 
+# The code snippet above shows an Array with 2 elements. Each element in an
+# Object Literal. Each of those Object Literals has 4 key-value pairs. This may LOOK
 # a bit daunting - it's OK! You don't need to be 100% comfortable with this, but it's
 # good to have some exposure before going into Mod 1.
 
 
-# YOU DO: Create an array of at least 3 EMAIL Object Literals, using the same 
+# YOU DO: Create an array of at least 3 EMAIL Object Literals, using the same
 # key-value pairs you used in your email Object above.
-# Then, log the email Array to the console.
\ No newline at end of file
+# Then, log the email Array to the console.
+
+email = [
+  {
+    sender: "Haewon",
+    receiver: "Stella",
```

Comment:
![it's me!](https://media4.giphy.com/media/2A8UoqgYXlFf1XxHDE/200.gif?cid=5a38a5a2hkbssnfgkjodslyqd74g71sdvdip3jfnop87lmpm&rid=200.gif&ct=g)